src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const order of actions) {
+    switch (true) {
+      case order.type === 'addProperties':
+        Object.assign(state, order.extraData);
+        break;
+
+      case order.type === 'removeProperties':
+        for (const toBeRemoved of order.keysToRemove) {
+          if (toBeRemoved in state) {
+            delete state[toBeRemoved];
+          }
+        }
+        break;
+
+      default:
+        for (const eraseAll in state) {
```

Comment:
`erase` is a verb, so it is a good name for a function, not for an object key

Reply:
Good advice !  Taken into account, correction applied.